packages/editor-ui/src/composables/useCopyPaste.ts

```diff
@@ -0,0 +1,266 @@
+import { onBeforeUnmount, onMounted, ref } from 'vue';
+import { debounce } from 'lodash-es';
+
+/**
+ * Determine user's browser type
+ * @source https://www.lucidchart.com/techblog/2014/12/02/definitive-guide-copying-pasting-javascript/
+ */
+const isSafari =
+	navigator.appVersion.search('Safari') !== -1 &&
+	navigator.appVersion.search('Chrome') === -1 &&
+	navigator.appVersion.search('CrMo') === -1 &&
+	navigator.appVersion.search('CriOS') === -1;
+const isInternetExplorer =
+	navigator.userAgent.toLowerCase().indexOf('msie') !== -1 ||
+	navigator.userAgent.toLowerCase().indexOf('trident') !== -1;
+
+type ClipboardEventFn = (data: string, event?: ClipboardEvent) => void;
+
+export function useCopyPaste(
```

Comment:
Can we rename this to `useClipboard` instead?

Reply:
Done, great suggestion!